packages/open-scd/src/editors/cleanup/datatypes-container.ts

```diff
@@ -470,12 +470,12 @@ export class CleanupDataTypes extends LitElement {
     return html`
       <div>
         <h1>
-          ${translate('cleanup.unreferencedDataTypes.title')}
+          ${this.getAttribute('cleanup.unreferencedDataTypes.title')}
```

Comment:
```suggestion
          ${get('cleanup.unreferencedDataTypes.title')}
```
Should be changed to get. It would solve 1 error from the snapshot tests that go wrong, but unfortunatly not all of them